.packit.yaml

```diff
@@ -56,16 +56,17 @@ jobs:
       - fedora-38-aarch64
       - fedora-38-ppc64le
       - fedora-38-x86_64
-      - centos-stream-9-aarch64
-      - centos-stream-9-ppc64le
-      - centos-stream-9-x86_64
+      - epel-9-aarch64
+      - epel-9-ppc64le
+      - epel-9-x86_64
 
   - job: tests
     trigger: pull_request
     identifier: e2e-tiers
     tmt_plan: /plans/e2e/tier-0
     targets:
-    - centos-stream-9-x86_64
+      epel-9-x86_64:
+        distros: [centos-stream-9]
```

Comment:
I think you can (should) remove line 69. With line 69 I suspect it's using a pure-centos-stream and not centos-stream with epel enabled.